lib/engine/round/operating.rb

```diff
@@ -308,6 +308,12 @@ def start_operating
       def change_entity(_action)
         return unless @current_entity.passed?
 
+        unless @current_entity.operating_info(@game.turn, @round_num)
```

Comment:
@current_entity.operating_history[[@game.turn, @round_num]] ||= OperatingInfo.new([], Action::Dividend.new(@game.current_entity, 'withhold'), 0)

Reply:
See above comment, I'm happy to change it if we get rid of the method call.